src/components/Home/Home.js

```diff
@@ -326,114 +336,115 @@ const Home = () => {
+                  margin='normal'
+                  id='date-picker-dialog'
+                  format='dd-MM-yyyy'
                   value={selectedDate}
                   onChange={handleDateChange}
-                  className="input"
+                  className='input'
                 />
               </MuiPickersUtilsProvider>
             </div>
           ) : null}
 
-          {toSearchValue === "Find By PinCode & Date" ? (
-            <div className="home_selectedPin">
-              <div className="home_selectedpincontainer">
+          {toSearchValue === 'Find By PinCode & Date' ? (
+            <div className='home_selectedPin'>
+              <div className='home_selectedpincontainer'>
                 <TextField
-                  id="outlined-number"
-                  margin="normal"
-                  label="Pin Code"
-                  type="number"
-                  variant="outlined"
-                  className="textField"
+                  id='outlined-number'
+                  margin='normal'
+                  label='Pin Code'
+                  type='number'
+                  variant='outlined'
+                  className='textField'
                   value={pin}
-                  onChange={(e) => setPin(e.target.value)}
+                  onChange={(e) => {
+                    setPinCodeSearch(false);
+                    setPin(e.target.value);
+                  }}
                 />
                 <SearchIcon
                   onClick={fetchDataUsingByPin}
                   style={{
-                    background: "#3f51b5",
-                    color: "#fff",
+                    background: '#3f51b5',
+                    color: '#fff',
                     padding: 5,
-                    cursor: "pointer",
+                    cursor: 'pointer',
                     width: 30,
                     height: 45,
                     marginTop: 16,
-                    borderRadius: "0 5px 5px 0",
+                    borderRadius: '0 5px 5px 0',
                   }}
-                  fontSize="medium"
+                  fontSize='medium'
                 />
               </div>
               <MuiPickersUtilsProvider utils={DateFnsUtils}>
                 <KeyboardDatePicker
-                  margin="normal"
-                  id="date-picker-dialog"
-                  format="dd-MM-yyyy"
+                  margin='normal'
+                  id='date-picker-dialog'
+                  format='dd-MM-yyyy'
                   value={selectedDate}
                   onChange={handleDateChange}
-                  className="input"
+                  className='input'
                 />
               </MuiPickersUtilsProvider>
             </div>
           ) : null}
 
-          {vaccineData.length === 0 ? null : (
-            <>
-              <VaccineDataMain vaccineData={currentVaccine} />
-              {pageNumber.length === 1 ? null : (
-                <Pagination
```

Comment:
@Yash-271120 This is required for pagination! please add this!

Thanks

Reply:
@Justinnn07 I again cloned the repo. This time I made the changes and just committed the changed files. No lock files this time.
[New commits](https://github.com/stephin007/Cowin-Vaccine-Availablity-Checker/pull/89/commits/503a6f792269e939fc45258187ac8519b89846b6)
I am new to github so was little bit confused.